rust/cardano-chain-follower/src/follow.rs

```diff
@@ -102,7 +101,7 @@ impl ChainFollower {
                     self.previous = self.current.clone();
                     // debug!("Post Previous update 3 : {:?}", self.previous);
                     self.current = next.point();
-                    self.fork = 0; // Mithril Immutable data is always Fork 0.
+                    self.fork = 0.into(); // Mithril Immutable data is always Fork 0.
```

Comment:
```suggestion
                    self.fork = Fork::IMMUTABLE; // Mithril Immutable data is always Fork 0.
```

Suggested constants added to the `Fork` type.